src/transformers/convert_slow_tokenizer.py

```diff
@@ -1560,6 +1560,21 @@ def tokenizer(self):
         tokenizer = Tokenizer(BPE(vocab_scores, merges, fuse_unk=False))
         if hasattr(tokenizer.model, "ignore_merges"):
             tokenizer.model.ignore_merges = True
+        num_reserved_special_tokens = 256
+        special_tokens = [
+            "<|begin_of_text|>",
+            "<|end_of_text|>",
+            "<|reserved_special_token_0|>",
+            "<|reserved_special_token_1|>",
+            "<|reserved_special_token_2|>",
+            "<|reserved_special_token_3|>",
+            "<|start_header_id|>",
+            "<|end_header_id|>",
+            "<|reserved_special_token_4|>",
+            "<|eot_id|>",
+            "<|python_tag|>",  # end of turn
+        ] + [f"<|reserved_special_token_{i}|>" for i in range(5, num_reserved_special_tokens - 5)]
```

Comment:
(let's remove them by default no?)

Reply:
We can create classes such as `Llama3TikTokenConverter(TikTokenConverter)` where special_tokens are set? How should we add them instead?